src/conv/invokers/impl_gemm.cpp

```diff
@@ -50,27 +50,60 @@ InvokerFactory MakeImplGemmDataInvokerFactory(const ConvolutionContext& ctx)
                     kernel.GetName() == "gridwise_convolution_backward_data_implicit_gemm_v1r1_ncdhw_kczyx_nkdhw"))
                 // clang-format on
                 {
-                    float zero = 0.f;
-                    TensorDescriptor workspaceDesc(
-                        miopenFloat, tensors.outDesc.GetLengths(), tensors.outDesc.GetStrides());
-                    SetTensor(handle, workspaceDesc, workSpace, &zero);
-                    if(handle.IsProfilingEnabled())
-                        elapsed += handle.GetKernelTime();
+                    const auto y           = tensors.wDesc.GetLengths()[2];
+                    const auto x           = tensors.wDesc.GetLengths()[3];
+                    const auto stride_h    = conv.GetConvStrides()[0];
+                    const auto stride_w    = conv.GetConvStrides()[1];
+                    const auto dialation_h = conv.GetConvDilations()[0];
```

Comment:
Typo

Reply:
Not sure about the location of typo here.

Reply:
```
dialation
  ^^^
```